src/devtools/client/webconsole/components/Output/message-types/ConsoleCommand.js

```diff
@@ -31,7 +33,7 @@ export default function ConsoleCommand(props) {
     source,
     type,
     level,
-    topLevelClasses: [],
+    topLevelClasses: topLevelClassName ? [topLevelClassName] : [],
```

Comment:
What is `[].compact()` ? Do you mean something like `[...].filter(i => i)`?

I have a slight preference to avoid unnecessarily using array manipulation methods.